server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.test.ts

```diff
@@ -970,7 +971,7 @@ describe('AgenticChatController', () => {
             assert.deepStrictEqual(chatResult, utils.createAuthFollowUpResult('full-auth'))
         })
 
-        it('returns a ResponseError if response streams return an error event', async () => {
+        it.only('returns a ResponseError if response streams returns an error event', async () => {
```

Comment:
I assume `.only` was committed by accident?

Reply:
yes, good catch. I'm used to the VSC repo that fails the build on these.